### PR TITLE
feat: Code action that inserts evr when missing ref in CalcPvr

### PR DESCRIPTION
**Pull request by vetlek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,16 @@
...
+        },
+        "septic.codeActions.insertEvrPosition": {
+          "scope": "window",
+          "type": "string",
+          "enum": [
+            "top",
+            "bottom"
+          ],
+          "default": "bottom",
+          "description": "Position of inserted evr. Bottom = Above CalcModl Top = Bellow DmmyAppl"
```
Comment (einarsi) [inline package.json:73]: "Evr" capitalised
"Below"
And while I'm at it: A comma or semi-colon after "CalcModl" wouldn't be wrong.

File: server/src/language-service/codeActionProvider.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    offset: number,
+    doc: ITextDocument
+): WorkspaceEdit {
+    let editBuilder = new WorkspaceEditBuilder();
+    let text = getTextEvr(name);
+    editBuilder.insert(doc.uri, doc.positionAt(offset), text);
+    return editBuilder.getEdit();
+}
+
+function getTextEvr(name: string) {
```
Comment (einarsi) [inline server/src/language-service/codeActionProvider.ts:133]: Possible to reuse snippet to keep code DRY?

**Review by einarsi — COMMENTED** [review]
Very good! Just a couple of extremely minor comments.

File: server/src/language-service/codeActionProvider.ts (hunk 2 above)
Comment (vetlek) [reply server/src/language-service/codeActionProvider.ts:133]: I don't think it is possible to reuse the snippet directly. The current version is copy paste from the snippet with the snippet specific things removed.

**Review by einarsi — APPROVED** [review]

**Pull request merged**